Core/tests/Unit/LongRunning/Response.php

```diff
@@ -0,0 +1,76 @@
+<?php
```

Comment:
We don't usually create additional files for unit test supporting classes. Please just put this class at the bottom of `OperationResponseTraitTest`. You'll need to wrap it in `@codingStandardsIgnoreStart` / `@codingStandardsIgnoreEnd` annotations to keep the code style check from complaining.

Same for `Value`.